yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -2152,3 +2158,46 @@ func validateSourceDBIOPSForAssessMigration() error {
 
 	return nil
 }
+
+// anonymizeIssueDetails anonymizes sensitive information in the details map
+// Currently handles FK_COLUMN_NAMES, can be extended for other sensitive keys
+func anonymizeIssueDetails(details map[string]interface{}) map[string]interface{} {
+	if details == nil {
+		return nil
+	}
+
+	anonymizedDetails := make(map[string]interface{})
+
+	for key, value := range details {
+		if key == queryissue.FK_COLUMN_NAMES {
+			if strValue, ok := value.(string); ok && strValue != "" {
+				// Split the string by comma to handle multiple FK columns
+				columnNames := strings.Split(strValue, ",")
```

Comment:
nit: we can just store it as a list instead of a string, no? Would be easier to process here.